src/Symfony/Bundle/TwigBundle/Extension/ActionsExtension.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Symfony\Bundle\TwigBundle\Extension;
 
+trigger_error('The '.__NAMESPACE__.'\ActionsExtension class is deprecated since version 2.2 and will be removed in Symfony 3.0.', E_USER_DEPRECATED);
```

Comment:
this is wrong, because it would trigger it all the time (TwigBundle is registering the extension in Twig)